src/login/LoginView.js

```diff
@@ -46,6 +46,9 @@ export class LoginView {
 	targetPath: string;
 	mailAddress: Stream<string>;
 	password: Stream<string>;
+	// we save the password dom element so that we can focus it when loginWith is passed as a param
+	// lazy because it wont have been initialized in oncreate
+	passwordInput: lazy<HTMLInputElement>;
```

Comment:
You want deferred, not lazy here. Right now it's just null by default and you will try to call it and then when you have it you wrap it into function